hoomd/md/EvaluatorPairExpandedGaussian.h

```diff
@@ -0,0 +1,218 @@
+// Copyright (c) 2009-2023 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+#ifndef __PAIR_EVALUATOR_EXPANDEDGAUSSIAN_H__
+#define __PAIR_EVALUATOR_EXPANDEDGAUSSIAN_H__
+
+#ifndef __HIPCC__
+#include <string>
+#endif
+
+#include "hoomd/HOOMDMath.h"
+
+/*! \file EvaluatorPairEXPANDEDGAUSSIAN.h
+    \brief Defines the pair evaluator class for shifted Gaussian potentials
+*/
+
+// need to declare these class methods with __device__ qualifiers when building in nvcc
+// DEVICE is __host__ __device__ when included in nvcc and blank when included into the host
+// compiler
+#ifdef __HIPCC__
+#define DEVICE __device__
+#define HOSTDEVICE __host__ __device__
+#else
+#define DEVICE
+#define HOSTDEVICE
+#endif
+
+namespace hoomd
+    {
+namespace md
+    {
+//! Class for evaluating the expanded Gaussian pair potential
+/*! <b>General Overview</b>
+
+    See EvaluatorPairLJ
+
+    <b>Gauss specifics</b>
+
+    EvaluatorPairExpandedGaussian evaluates the function:
+    \f[ V_{\mathrm{expanded_gauss}}(r) = \varepsilon \exp \left[ -
+    \frac{1}{2}\left( \frac{r-\delta}}{\sigma}\right)^2 \right] \f]
+
+    The expanded Gaussian potential does not need diameter or charge. Three
+    parameters are specified and stored in a Scalar3.
+    \a epsilon is placed in \a params.x, \a sigma is in \a params.y, and
+    \a delta is in \a params.z.
```

Comment:
```suggestion
    The expanded Gaussian potential does not need diameter or charge.
```
Remove inaccurate comment.